concise-mid-tag.cddl

```diff
@@ -74,22 +74,26 @@ identity-triple-record = [
   [ + verification-key-map ] ; DevID, or semantically equivalent
 ]
 
-pkix-base64-key-type = tstr
-pkix-base64-cert-type = tstr
+$rawkey-type-choice /= tagged-pkix-base64-key-type
+
+; Raw key in DER format base64-encoded.
+tagged-pkix-base64-key-type = #6.554(tstr)
+
+$authority-path-type-choice /= tagged-pkix-base64-cert-type
+
+; X.509 certificate chain in DER format base64-encoded
+tagged-pkix-base64-cert-type = #6.555(tstr)
 
 verification-key-map = {
-  ; Verification key in DER format base64-encoded.
-  ; Typically, but not necessarily a public key.
-  comid.key => pkix-base64-key-type
-  ; Optional X.509 certificate chain corresponding to the public key
-  ; in comid.key, encoded as an array of one or more base64-encoded
-  ; DER PKIX certificates.  The certificate containing the public key
-  ; in comid.key MUST be the first certificate.  This MAY be followed
-  ; by additional certificates, with each subsequent certificate
-  ; being the one used to certify the previous one.
-  ? comid.keychain => [ + pkix-base64-cert-type ]
+  raw-key-or-authority-path
```

Comment:
keychain was an array of cert-type. This change seems to have forgotten the array aspect.

Reply:
The array is what captures the path. The elements of the array are keys with authority. Maybe it is more intuitive to call them 'authority-key' in contrast to 'raw-key' which has no authority context? Then we either have a raw-key or an array of authority key.

Reply:
An authority-key could be a DID which is a network abstraction rather than a path abstraction. COSE_Sign is a recursion abstraction etc... The 'path' may be part of the underlying data structure behind the key. Having an array of authority-key is only needed for X.509 where the authority key abstraction is a certificate.

Reply:
> The array is what captures the path. The elements of the array are keys with authority. Maybe it is more intuitive to call them 'authority-key' in contrast to 'raw-key' which has no authority context? Then we either have a raw-key or an array of authority key.

good catch!